src/services/subscriptionService.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const db = require('../db/db');
+
+module.exports.subscribe = async email => await db.insertEmail(email);
+
+module.exports.isSubscribed = async email => await db.isEmailInDB(email);
```

Comment:
Тут не треба await'ити функції роботи з db, ти можешь просто їх повертати.

Вони повернуть promise, який ти вже й так await'иш у subscription controller.

```suggestion
module.exports.subscribe = db.insertEmail;

module.exports.isSubscribed = db.isEmailInDB;
```

Reply:
Погоджуюсь, але все ж таки, як на мене, краще повернути хоча б функції, які будуть делегувати роботу з бд, а не посилання, таким чином:
module.exports.subscribe = email => db.insertEmail(email);

module.exports.isSubscribed = email => db.isEmailInDB(email);